sound/soc/sof/intel/hda-loader.c

```diff
@@ -286,11 +285,11 @@ int hda_cl_cleanup(struct device *dev, struct snd_dma_buffer *dmab,
 
 	return ret;
 }
-EXPORT_SYMBOL_NS(hda_cl_cleanup, SND_SOC_SOF_INTEL_HDA_COMMON);
 
 #define HDA_CL_DMA_IOC_TIMEOUT_MS 500
 
-int hda_cl_copy_fw(struct snd_sof_dev *sdev, struct hdac_ext_stream *hext_stream)
+static int hda_cl_copy_fw(struct snd_sof_dev *sdev,
+			  struct hdac_ext_stream *hext_stream)
```

Comment:
that one is probably fine to remove, I don't recall that it's used by BPT/BRA.

Reply:
I will just drop this as it is.